backend/src/api/context.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
 /// The context that is accessible to every resolver in our API.
 pub(crate) struct Context {
     pub(crate) db: Transaction,
-    pub(crate) user: Option<User>,
+    pub(crate) authn: AuthContext,
```

Comment:
Why `authn`? Like I know "authz" and "authn" stand for authorization and authentification respectively, so that would be authentification... but. Why not just `auth` or `auth_context`? I find it a bit weird to explicitly exclude authorization here? The field is used for both.

Reply:
True, why not `auth`. :shrug: